### PR TITLE
OBSDOCS-333 - Logging 5.6.8 - Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.10, 4.11, 4.12

Issue: [OBSDOCS-333](https://issues.redhat.com//browse/OBSDOCS-333)

Link to docs preview: https://62326--docspreview.netlify.app/openshift-enterprise/latest/logging/v5_6/logging-5-6-release-notes.html#cluster-logging-release-notes-5-6-8_logging-5.6-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
LGTM

**Comment by libander** [comment]
@anpingli / @kabirbhartiRH - lgmt please

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Comment by anpingli** [comment]
LGTM

File: modules/logging-rn-5.6.8.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+* Before this update, when viewing logs within the {product-title} web console, clicking the *more data available* option loaded more log entries only the first time it was clicked. With this update, more entries are loaded with each click. (link:https://issues.redhat.com/browse/OU-187[OU-187])
+
+* Before this update, when viewing logs within the {product-title} web console, clicking the *streaming* option would only display the *streaming logs* message without showing the actual logs. With this update, both the message and the log stream are displayed correctly.(link:https://issues.redhat.com/browse/OU-189[OU-189])
+
+* Before this update, the Loki Operator reset errors in a way that made identifying configuration problems difficult to troubleshoot. With this update, errors persist until the configuration error is resolved.(link:https://issues.redhat.com/browse/LOG-4158[LOG-4158])
```
Comment (adellape) [inline modules/logging-rn-5.6.8.adoc:17]: Missing space after punctuation:

```suggestion
* Before this update, the Loki Operator reset errors in a way that made identifying configuration problems difficult to troubleshoot. With this update, errors persist until the configuration error is resolved. (link:https://issues.redhat.com/browse/LOG-4158[LOG-4158])
```

**Pull request merged**